src/ci/aflpp.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+mkdir -p artifacts/third-party/aflpp-linux
+
+sudo apt-get install -y llvm llvm-dev clang
+
+git clone https://github.com/AFLplusplus/AFLplusplus
+cd AFLplusplus
+# checkout v2.68c
+git checkout ee206da3897fd2d9f72206c3c5ea0e3fab109001
+make
+(test -d llvm_mode && cd llvm_mode && make)
+#(test -d gcc_plugin && cd gcc_plugin && make)
+(cd examples/libdislocator && make)
```

Comment:
libdislocator is in AFLplusplus, not in examples.   Additionally, the resulting file is in AFLplusplus, not AFLplusplus/examples/libdislocator

This line should be:

 `(cd libdislocator && make)`

Reply:
you would think I would know that ;)